webpack/components/extensions/RegistrationCommands/RegistrationCommandsPageHelpers.js

```diff
@@ -0,0 +1,21 @@
+// Activation Keys helpers
+
+export const validateAKField = (hostGroupId, userKeys, hgKeys) => {
+  if (hostGroupId === '') {
+    return (userKeys?.length > 0 ? 'success' : 'error');
```

Comment:
Can you use constants from foreman core for SUCCESS, ERROR?

Reply:
Which constants do you mean, I couldn't find any 'success' & 'error' constants. I need to have it in lowercase for the PF4 component `FormGroup` in `RegistrationCommands/fields/ActivationKeys.js`